dist/origin-web-common-ui.js

```diff
@@ -2315,7 +2277,7 @@ angular.module('openshiftCommonUI').provider('NotificationsService', function()
     };
 
     var clearNotifications = function () {
-      _.take(notifications, 0);
+      notifications.length = 0;
```

Comment:
Found a bug writing these tests! The previous `clearNotifications` implementation did nothing since `_.take` returns a new array.